deeppavlov/core/trainers/nn_trainer.py

```diff
@@ -297,5 +302,7 @@ def train(self, iterator: DataLearningIterator) -> None:
         else:
             log.warn(f'Using {self.__class__.__name__} for a pipeline without batched training')
 
+        # FIXME: Should we really save the model without clear improvement in score?
```

Comment:
Sometimes we would want to skip validation step altogether and just train a model for several epochs.

Reply:
OK, I've included a comment describing this situation.